common/eth2/src/lighthouse.rs

```diff
@@ -162,9 +167,45 @@ pub struct SystemHealth {
     pub misc_node_boot_ts_seconds: u64,
     /// OS
     pub misc_os: String,
+    /// Data directory path
+    pub data_dir: String,
 }
 
 impl SystemHealth {
+    /// Gets the network which should be used.
+    fn get_network() -> String {
+        // Try to get network from command line arguments
+        let args: Vec<String> = std::env::args().collect();
+        if let Some(pos) = args.iter().position(|x| x == "--network") {
+            if pos + 1 < args.len() {
+                return args[pos + 1].clone();
+            }
+        }
+
+        // Default to mainnet
+        "mainnet".to_string()
+    }
+
+    /// Gets the datadir which should be used.
+    fn get_data_dir() -> PathBuf {
+        // Try to get datadir from command line arguments
+        let args: Vec<String> = std::env::args().collect();
```

Comment:
We shouldn't be re-parsing the CLI args like this. They are already parsed by clap and available in the config